mapper/complementaryContentMapper.go

```diff
@@ -45,9 +53,9 @@ func (ccm *ComplementaryContentCPHMapper) mapToUppComplementaryContentUpdate(mpc
 		},
 		Type:                   contentType,
 		Brands:                 buildBrands(),
```

Comment:
I think the 'buildBrands' function should be moved to uppContentPlaceholder.go.

Reply:
moved.